reactor-core/src/main/java/reactor/core/publisher/FluxProcessor.java

```diff
@@ -43,7 +43,7 @@
  * @param <OUT> the output value type
  */
 public abstract class FluxProcessor<IN, OUT> extends Flux<OUT>
-		implements Processor<IN, OUT>, CoreSubscriber<IN>, Scannable, Disposable {
+		implements Processor<IN, OUT>, CoreSubscriber<IN>, Scannable, Disposable, Contextable {
```

Comment:
if the interface is only added to `FluxProcessor` and the new sink, maybe make its name more sink-oriented? like `SinkSerializable` for instance?

Reply:
I see no reason to make it sink-oriented, especially given that it can be useful in other places.
`SinkSerializable` sounds like "a Sink that supports Java Serialization" to me :D